core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditorSpec.js

```diff
@@ -124,4 +124,336 @@ describe('State Editor controller', function() {
       expect(scs.savedMemento.getHtml()).toEqual('This is some content.');
     });
   });
+
+  describe('TrainingDataService', function() {
```

Comment:
Why have you added this here? We have a separate Spec file for it `TrainingDataServiceSpec.js`, right?

Reply:
@shubha1593 I'm sorry, it wasn't intentional. I'm not sure how I ended up inserting it there... Removed it now.